spec/lib/mvi/service_spec.rb

```diff
@@ -249,8 +247,6 @@
 
     context 'when a MVI invalid request response is returned' do
       it 'should raise a invalid request error', :aggregate_failures do
-        invalid_xml = File.read('spec/support/mvi/find_candidate_invalid_request.xml')
-        allow_any_instance_of(MVI::Service).to receive(:create_profile_message).and_return(invalid_xml)
```

Comment:
This uses a VCR cassette, which records the request and the response. Therefore these two lines are are unnecessary.  Also removed the `find_candidate_invalid_request.xml` file